### PR TITLE
chore: bump istio chart and version

### PR DESCRIPTION
**Pull request by GoelDeepak**
**What type of PR is this?**
Chore

**What this PR does/ why we need it**:
Bumps istio chart and version

**Which issue(s) this PR fixes**:
https://jira.d2iq.com/browse/D2IQ-69875

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [x] The commit message explains the changes and why are needed.
* [x] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: addons/istio/1.6.x/istio-2.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: istio
+  labels:
+    kubeaddons.mesosphere.io/name: istio
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.6.3-2"
```
Comment (shaneutt) [inline addons/istio/1.6.x/istio-2.yaml:9]: This revision and the below appversion of istio appear to be disjointed, should this not be:
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.6.4-2"
```
?

Comment (GoelDeepak) [reply addons/istio/1.6.x/istio-2.yaml]: Done

File: addons/istio/1.6.x/istio-2.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    minSupportedVersion: v1.16.0
+  cloudProvider:
+    - name: aws
+      enabled: false
+    - name: azure
+      enabled: false
+    - name: docker
+      enabled: false
+    - name: none
+      enabled: false
```
Comment (shaneutt) [inline addons/istio/1.6.x/istio-2.yaml:36]:
```suggestion
      enabled: false
    - name: gcp
      enabled: false
```

File: addons/istio/1.6.x/istio-2.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    appversion.kubeaddons.mesosphere.io/kiali: "1.18.0"
+    appversion.kubeaddons.mesosphere.io/jaeger: "1.16.0"
+    stage.kubeaddons.mesosphere.io/kiali: Preview
+    stage.kubeaddons.mesosphere.io/jaeger: Preview
+    endpoint.kubeaddons.mesosphere.io/kiali: "/ops/portal/kiali"
+    endpoint.kubeaddons.mesosphere.io/jaeger: "/ops/portal/jaeger"
+    docs.kubeaddons.mesosphere.io/istio: "https://istio.io/docs/"
+    docs.kubeaddons.mesosphere.io/kiali: "https://istio.io/docs/tasks/telemetry/kiali/"
+    docs.kubeaddons.mesosphere.io/jaeger: "https://istio.io/docs/tasks/telemetry/distributed-tracing/jaeger/"
+    values.chart.helm.kubeaddons.mesosphere.io/istio: "https://raw.githubusercontent.com/mesosphere/charts/63bbc17eda76f2136f6ab4f6d6eef7e764e5f0a5/staging/istio/values.yaml"
```
Comment (shaneutt) [inline addons/istio/1.6.x/istio-2.yaml:20]: It looks like the values for Istio have changed considerably in recent releases of the chart:

https://raw.githubusercontent.com/mesosphere/charts/master/staging/istio/values.yaml

We should update this version with a hash that reflects the values present for `v1.6.4`.

File: addons/istio/1.6.x/istio-2.yaml (hunk 2 above)
Comment (GoelDeepak) [reply addons/istio/1.6.x/istio-2.yaml:38]: 👍

File: addons/istio/1.6.x/istio-2.yaml (hunk 3 above)
Comment (GoelDeepak) [reply addons/istio/1.6.x/istio-2.yaml]: 👍

**Comment by joejulian** [comment]
The e2e test that failed doesn't install istio, so I'm going to consider it an unrelated flake.

**Pull request merged**